templates/todo/common/infra/bicep/app/apim-api.bicep

```diff
@@ -0,0 +1,98 @@
+param name string
+
+@description('The name of the API')
+@minLength(1)
+param apiName string
```

Comment:
It would help if this was more descriptive.  What does the name of the API actually translate to?

Reply:
fixed